tiledb/sm/storage_manager/storage_manager.cc

```diff
@@ -2625,5 +2723,177 @@ Status StorageManagerCanonical::group_metadata_vacuum(
   return consolidator->vacuum(group_name);
 }
 
+void StorageManager::encryption_key_from_configs(EncryptionKey &enc_key, const Config* config) {
+  if (!config) {
+    config = &config_;
+  }
```

Comment:
NIT: add line breaks after closing bracket of if statement.

Reply:
ok